tests/test_grimgrains.py

```diff
@@ -0,0 +1,75 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers.grimgrains import GrimGrains
+from tests import ScraperTest
+
+
+class TestGrimGrainsScraper(ScraperTest):
+    scraper_class = GrimGrains
+
+    def test_host(self):
+        self.assertEqual("grimgrains.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Hundred Rabbits", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual("okonomiyaki", self.harvester_class.title())
+
+    def test_total_time(self):
+        self.assertEqual(20, self.harvester_class.total_time())
+
+    def test_yields(self):
+        self.assertEqual("4 servings", self.harvester_class.yields())
+
+    def test_image(self):
+        self.harvester_class.url = "https://grimgrains.com/site/okonomiyaki.html"
```

Comment:
Note-to-future: this `url` attribute is set because we return a fully-qualified `image` URL, and when the tests run (on HTML, outside of a URL context) the base URL is `test.example.com`, which looks wrong.